packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -1,3 +1,16 @@
+/*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-navigation-logo-background-color: Specifies the background color of the component.
+* @prop --calcite-navigation-logo-background-color-hover: Specifies the background color of the component when hovered.
+* @prop --calcite-navigation-logo-background-color-press: Specifies the background color of the component when active.
```

Comment:
this component shouldn't need state tokens. If necessary move the pseudoselectors to host. See [Button](https://github.com/Esri/calcite-design-system/pull/10358) for examples.